tests/cases/fourslash/commentBraceCompletionPosition.ts

```diff
@@ -15,9 +15,14 @@
 
 goTo.marker('1');
 verify.isValidBraceCompletionAtPosition('(');
+verify.not.isValidBraceCompletionAtPosition('"');

```

Comment:
Would it be worthwhile to test a position that isn't at EOL (e.g. a contraction, as in the bug report)?